.gitignore

```diff
@@ -3,3 +3,4 @@ dist
 docs
 .DS_Store
 .pnpm-debug.log*
+tests/config.js
```

Comment:
I added `config.ts` in `tests/.gitignore` already.

Reply:
done